statsd/statsd.go

```diff
@@ -452,6 +454,7 @@ func newWithWriter(w Transport, o *Options, writerName string) (*Client, error)
 		telemetry:             &statsdTelemetry{},
 		errorOnBlockedChannel: o.channelModeErrorsWhenFull,
 		errorHandler:          o.errorHandler,
+		originDetection:       isOriginDetectionEnabled(o),
```

Comment:
`isOriginDetectionEnabled` returns false if containerID is set. Is it okay if we don't send external env in this case?